File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    //prompt for int between 1 & 8 inclusive
    do
    {
        n = get_int("Height:");
    }
    while (n < 1 || n > 8);
    for (int i = 0; i < n; i++)
    {
        for (int j = 1 + i; j < n; j++)
        {
            printf(" ");
        }
        for (int k = 0; k < 1 + i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: mario.py
from cs50 import get_int

# Get height 1-9 from user
while True:
    h = get_int("Height: ")
    if h > 0 and h < 10:
        break
    
# Print Block
for i in range(1, h + 1):
    print(" " * (h - i), end="")
    print("#" * i)